cpp/ps0000/ps0000_soln.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream> // 2/2
+
+using namespace std;
+
+int main() // 1/1
+{
+  cout << "Hello, World!\n"; // 2/4
```

Comment:
- [ ] used `std::cout <<` for outputting instead of `cout >>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line